### ISSUE TITLE
Remove Leaderboards(?)

### ISSUE THREAD

**Issue by benjaminpjones** [online-go/online-go.com#1789]
**Is your feature request related to a problem? Please describe.**
It seems to be a relatively popular opinion in the Forums that Leaderboards could either use improvement or be removed altogether.  The major reasons being that:

- The algorithm is opaque (it is documented in the FAQ, but it is not clear from page itself what is being reported)
- The algorithm may not work as stated.
- It is taking up valuable real-estate in the nav bar

I am happy to submit a PR that removes the page, but would like confirmation that this is desirable.

**Describe the solution you'd like**
Remove the Leaderboards page from the menu (and from OGS altogether)

**Describe alternatives you've considered**
- Fix the Leaderboards page or make it more clear what numbers are being reported
    - Pros: we'll have a more useful leaderboard
    - Cons: Requires development effort
- Remove the Leaderboards page from the menu, but accessible elsewhere
    - Pros: less drastic action
    - Cons: more code to maintain, not much benefit

**Additional context**
- https://forums.online-go.com/t/fix-or-remove-ogs-tournament-leaderboards/36028
- https://forums.online-go.com/t/leaderboards-what-do-the-gray-numbers-mean/33224 (disclaimer: I made this post)
- https://forums.online-go.com/t/ogs-design-update/42831/17 (I made this comment, but many supporting comments follow)

**Comment by anoek** [comment]
Yeah, I reckon we should just remove it for the time being. I'd like to have something like that that's useful and interesting at some point, but that'd be revamping it anyways, not much use in saving the little ui bits we have now .

**Comment by benjaminpjones** [comment]
Sounds good.  I'll wait for `react18` to get merged in (else a merge conflict in routes.tsx) to submit a PR.  Also happy to help with a revamp!
